rllib/connectors/util.py

```diff
@@ -81,45 +77,6 @@ def get_action_connectors_from_config(
     return ActionConnectorPipeline(ctx, connectors)
 
 
-@PublicAPI(stability="alpha")
-def create_connectors_for_policy(policy: "Policy", config: TrainerConfigDict):
```

Comment:
Hey, sorry for not being clear about that. The earlier comment on this from jun was deleted because the line was removed:
It's policy.init_connectors() and jun wanted me to not do deprecation warnings here at all and just remove it.

Reply:
nvm. understood.